apps/web/tsconfig.json

```diff
@@ -1,5 +1,51 @@
 {
   "extends": "tsconfig/nextjs.json",
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
-  "exclude": ["node_modules"]
+  "compilerOptions": {
+    "plugins": [
+      {
+        "name": "next"
+      }
+    ],
+    "strictNullChecks": true,
+    "paths": {
+      "@/*": [
+        "./src/*"
+      ],
+      "@/api/*": [
+        "./src/api/*"
+      ],
+      "@/app/*": [
+        "./src/app/*"
+      ],
+      "@/components/*": [
+        "./src/components/*"
+      ],
+      "@/common/*": [
+        "./src/components/common/*"
+      ],
+      "@/elements/*": [
+        "./src/components/elements/*"
+      ],
+      "@/configs/*": [
+        "./src/configs/*"
+      ],
+      "@/pages/*": [
+        "./src/pages/*"
+      ],
+      "@/views/*": [
+        "./src/views/*"
+      ]
+    },
+    "allowJs": true,
```

Comment:
you have the allowJS and incremental in the nextjs.json in tsconfig folder, so we don't need them here.